src/list.json

```diff
@@ -172,7 +172,14 @@
         "link": "https://github.com/pusher/hacktoberfest/blob/main/README.md"
       }
     ],
-    "Q": [],
+    "Q": [{
+                "organization": "QuestDB",
```

Comment:
@ShivayeModi Please Re-style it via pasting this command in the Repo Directory.
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2338163/patch | git am`
`git push`